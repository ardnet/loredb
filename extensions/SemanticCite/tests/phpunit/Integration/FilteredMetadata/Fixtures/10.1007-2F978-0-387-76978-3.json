{"subtitle":[],"issued":{"date-parts":[[2011]]},"score":1.0,"prefix":"http:\/\/id.crossref.org\/prefix\/10.1007","container-title":null,"reference-count":0,"deposited":{"date-parts":[[2013,9,6]],"timestamp":1378425600000},"title":"The Little Black Book of Neuropsychology","editor":[{"affiliation":[],"family":"Schoenberg","given":"Mike R."},{"affiliation":[],"family":"Scott","given":"James G."}],"type":"book","DOI":"10.1007\/978-0-387-76978-3","ISBN":["http:\/\/id.crossref.org\/isbn\/978-0-387-70703-7","http:\/\/id.crossref.org\/isbn\/978-0-387-76978-3"],"URL":"http:\/\/dx.doi.org\/10.1007\/978-0-387-76978-3","source":"CrossRef","publisher":"Springer Science + Business Media","indexed":{"date-parts":[[2015,6,11]],"timestamp":1434026028914},"member":"http:\/\/id.crossref.org\/member\/297"}