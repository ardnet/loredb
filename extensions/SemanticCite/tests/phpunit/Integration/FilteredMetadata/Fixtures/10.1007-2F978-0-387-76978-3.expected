{{#scite:
 |reference=schoenberg2013th
 |type=book
 |title=The Little Black Book of Neuropsychology
 |editor=Mike R. Schoenberg;James G. Scott|+sep=;
 |publisher=Springer Science + Business Media
 |year=2013
 |doi=10.1007/978-0-387-76978-3
}}